src/components/CookForm/styles.scss

```diff
@@ -18,7 +18,7 @@
 }
 
 .twoColSection {
-  display: flex;
+  display: block;
```

Comment:
когда Flex  был — все поля в одну строку вмещались, с блоком, каждое поле друг под другом.
> 18 апр. 2016 г., в 14:21, Rodion <notifications@github.com> написал(а):
> 
> In src/components/CookForm/styles.scss <https://github.com/sars/appetini-front/pull/28#discussion_r60041711>:
> 
> > @@ -18,7 +18,7 @@
> >  }
> >  
> >  .twoColSection {
> > -  display: flex;
> > +  display: block;
> как я понимаю это ведет к тому что будут имя и фамилия на разных строках в половину ширины. 
> почему поменял? зачем так?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/sars/appetini-front/pull/28/files/1466e0fe003b339f2577c5f774caf0dd296db985#r60041711>